orttraining/orttraining/python/training/onnxblock/_building_blocks.py

```diff
@@ -0,0 +1,146 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# _building_blocks.py
```

Comment:
Oh, I generally add the name so that if I do a global search the file will pop up in vs code. I don't think there is any need from guidance perspective for this file to have the name of the file in it. If you feel is not necessary, I can remove it.

Reply:
that is fine.